### PR TITLE
Search improvement: Strip whitespace again after splitting name

### PR DESCRIPTION
**Pull request by VirginiaDooley**
Closes https://github.com/DemocracyClub/yournextrepresentative/issues/2048

Before
Screenshot 2023-04-26 at 4 03 55 PM

After
Screenshot 2023-04-26 at 4 04 13 PM

### CODE REVIEW

**Review by Bekabyx — APPROVED** [review]

**Comment by symroe** [comment]
This is a good fix for the symptom, but it doesn't prevent the underlying problem in the issue.

Can you look at the places where we can add or edit names (bulk adding, person add/edit form, change name form) and make sure they're using https://github.com/DemocracyClub/yournextrepresentative/blob/f65b482419bf29d01831f3e1f1aac364531635a0/ynr/apps/people/forms/fields.py#L93?

And can you add test cases for adding a name with spaces and making sure what's saved in the database doesn't contain spaces?

**Comment by VirginiaDooley** [comment]
[Improve person search results: whitespace](https://trello.com/c/SlqHWc9s/3337-improve-person-search-results-whitespace)

File: ynr/apps/bulk_adding/forms.py [hunk 1]
```diff
@@ -240,7 +244,7 @@ def clean(self):
 
 
 class NameOnlyPersonForm(forms.Form):
-    name = forms.CharField(
+    name = StrippedCharField(
```
Comment (symroe) [inline ynr/apps/bulk_adding/forms.py:247]: Just checking, does this require a migration?

Comment (VirginiaDooley) [reply ynr/apps/bulk_adding/forms.py:247]: I've checked again, it does not.

**Pull request merged**